imports/plugins/included/product-variant/components/productGrid.js

```diff
@@ -81,6 +82,26 @@ class ProductGrid extends Component {
     }
   }
 
+  renderFilteredCount() {
+    const { selectedProductIds } = this.props;
+    const count = selectedProductIds.length;
+    const filterByProductIds = Session.get("filterByProductIds");
+
+    if (filterByProductIds) {
+      return (
+        <div style={{ paddingTop: "24px", paddingLeft: "8px" }}>
```

Comment:
Instead of hard coding the values here (and on line 96), we should use `withStyles`, and put these styles in a class, and then use our `theme.spacing` unites to determine these values. One spacing unit is `8px`, so you'd do something like this:

```
import withStyles from "@material-ui/core/styles/withStyles";

const styles = (theme) => ({
  filterContainer: {
    paddingLeft: theme.spacing()
    paddingTop: theme.spacing(3)
  }
});

...

<div className={classes.filterContainer}>
```